packages/xod-project/src/constants.js

```diff
@@ -22,11 +22,17 @@ export const PIN_TYPE = {
   T3: 't3',
 };
 
+export const INPUT_PULSE_PIN_BINDING_OPTIONS = {
+  Never: 'Never',
```

Comment:
What's wrong with all-caps keys?
IMHO, `INPUT_PULSE_PIN_BINDING_OPTIONS['On Boot']` would look a bit awkward

Reply:
Yep. But in this case, we don't need to have another map that will point from 'On Boot' literal to 'ON_BOOT' key in this map. Or not? 🤔